config/special_users.json

```diff
@@ -7,5 +7,6 @@
     "U01230GJXS6",
     "UE8DH0UHM",
     "U015MCCBXBP",
-    "U011CFN98K1"
+    "U011CFN98K1",

```

Comment:
Remove the comma for properly formatted JSON

Reply:
Resolving as the comma is needed for slackbot.

Reply:
Please see comment above.

Reply:
Please do not unresolve this comment again as it is needed for the above.